common/server_response_containers.cpp

```diff
@@ -41,8 +41,10 @@ void GameEventStorage::setGameEventContext(const ::google::protobuf::Message &_g
 void GameEventStorage::enqueueGameEvent(const ::google::protobuf::Message &event,
                                         int playerId,
                                         GameEventStorageItem::EventRecipients recipients,
-                                        int _privatePlayerId)
+                                        int _privatePlayerId,
+                                        bool _overwriteOwnership)
```

Comment:
on looking at the way setForcedByJudge is implemented I'd like to see this implemented consistently, please remove _overwriteOwnership from the enqueGameEvent function and instead have cmdRevealCards use ges.setOverwriteOwnership